service/history/execution/mutable_state_builder.go

```diff
@@ -149,6 +149,8 @@ type (
 		insertReplicationTasks  []persistence.Task
 		insertTimerTasks        []persistence.Task
 
+		workflowRequests map[persistence.WorkflowRequest]struct{}
```

Comment:
`CopyToPersistence()` is not updated. should it be?

Reply:
it shouldn't. workflow requests are treated like transfer/timer tasks

Reply:
may I suggest adding a comment to the CopyToPersistence() if they're to be excluded specifically. Someone like me lacking context would probably go 'fix' that and cause more problems